src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/templates/layouts/Title.tsx

```diff
@@ -0,0 +1,10 @@
+export interface TitleProps {
+    text: string;
+}
+
+/**
+ * @unreleased
+ */
+export default function Title({text}: TitleProps) {
```

Comment:
I'm actually not sure about the Form prefix for title and description.  Technically this is not the form title because this is a separate setting that lives inside the design settings.  In fact the setting is labeled "heading" so that it's clear it's not the actual form title but just a heading that _can_ be displayed above your form. 🤔

Reply:
Yeah... this is **not** the form title, but rather a generic design setting (which makes naming difficult).

Reply:
I think @kjohnson's suggestion makes the most sense but as `HeaderTitle` and `HeaderDescription`.  Reason being these components live inside the `Header` component and surfaced in the api within _layouts_ .  Just like the `FieldLabel` layout is prefixed with `Field` because it lives inside the `Field`.

Reply:
Jason: 0
Kyle: 1

Reply:
resolved 619eb9fd